[DOCS] severity tagging (#11354)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/expectations/manage_expectations.md

4. Select a data quality issue to test for or an option for writing your own test.

   :::tip Options for accelerating test coverage

>    If you selected the **Schema**, **Volume**, or **Completeness** data quality issue, you will have the **Automatic** option to generate Expectations for [Anomaly Detection](/cloud/overview/accelerating_test_coverage.md#anomaly-detection). Generated Expectations will be assigned a failure severity level of warning, which you can edit later. If you instead want to create your own rules, click **Manual**.

Reviewer: Similar grammar/language consistency nit. In this section the word `severity level of X` is being used, whereas above it was just `X severity`

Author: ```suggestion
   If you selected the **Schema**, **Volume**, or **Completeness** data quality issue, you will have the **Automatic** option to generate Expectations for [Anomaly Detection](/cloud/overview/accelerating_test_coverage.md#anomaly-detection). Generated Expectations will default to warning severity, which you can edit later. If you instead want to create your own rules, click **Manual**.
```

Keep the following requirements in mind when working with ExpectAI:
- Your organization must be using a [fully-hosted deployment](/cloud/deploy/deployment_patterns.md).
- The Data Asset's Data Source must be AlloyDB, Amazon Aurora PostgreSQL, Citus, Databricks SQL, Neon, PostgreSQL, Redshift, or Snowflake.

> - Generated Expectations will be assigned a failure severity level of warning, which you can edit later.

Author: ```suggestion
- Generated Expectations will default to warning severity, which you can edit later.
```
